validator/rpc/wallet_test.go

```diff
@@ -304,3 +304,59 @@ func Test_writeWalletPasswordToDisk(t *testing.T) {
 	err = writeWalletPasswordToDisk(walletDir, "somepassword")
 	require.NotNil(t, err)
 }
+
+func createImportedWalletWithAccounts(t testing.TB, numAccounts int) (*Server, [][]byte) {
+	localWalletDir := setupWalletDir(t)
+	defaultWalletPath = localWalletDir
+	ctx := context.Background()
+	w, err := accounts.CreateWalletWithKeymanager(ctx, &accounts.CreateWalletConfig{
+		WalletCfg: &wallet.Config{
+			WalletDir:      defaultWalletPath,
+			KeymanagerKind: keymanager.Imported,
+			WalletPassword: strongPass,
+		},
+		SkipMnemonicConfirm: true,
+	})
+	require.NoError(t, err)
+	//require.NoError(t, w.SaveHashedPassword(ctx))
+	km, err := w.InitializeKeymanager(ctx, iface.InitKeymanagerConfig{ListenForChanges: false})
+	require.NoError(t, err)
+	ss := &Server{
```

Comment:
`ss` is a weird name. Why not `s` or `server`?

Reply:
changed to s